health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/UpdateBoundary.js

```
@@ -600,45 +600,8 @@ const UpdateBoundary = ({hierarchyData }) => {
-          setShowToast({ key: "error", label: "DELIVERY_CYCLE_MISMATCH_LENGTH_ERROR" });
-          return false;
-        }
-        if (isDeliveryError === false) {
-          setShowToast({ key: "error", label: "DELIVERY_RULES_ERROR" });
-          return false;
-        }
-        setShowToast(null);
-        return true;
-        case "DataUploadSummary":
+      case "summary":
+        const updateBoundary = restructureBoundaryData(totalFormData?.HCM_CAMPAIGN_SELECTING_BOUNDARY_DATA?.boundaryType?.selectedData);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)